crates/starknet-server/tests/test_advancing_time.rs

```diff
@@ -133,7 +283,7 @@ mod advancing_time_tests {
         // set time and assert if it's greater/equal than future_time, check if it's inside buffer
         // limit
         let now = get_unix_timestamp_as_seconds();
-        let future_time = now + 100;
+        let future_time = now + 1000;
```

Comment:
Why the increase?

Reply:
Explained on the call

Reply:
comment added in 5b8dba1ad980cd3310fd4a14ed78eb636e12b8ea